MediaAdminBundle/Resources/translations/messages.en.yml

```diff
@@ -1,3 +1,14 @@
+open_orchestra_backoffice:
```

Comment:
That's why those translations were broken: the primary node (open_orchestra_backoffice) is automatically generated in the twig, so it must be open_orchestra_backoffice.
But the block is described in the mediaAdmin bundle, so the translations must be there too.